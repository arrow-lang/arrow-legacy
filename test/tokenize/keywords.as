and
or
xor
not
let
def
mutable
true
false
if
while
loop
break
continue
return
global
