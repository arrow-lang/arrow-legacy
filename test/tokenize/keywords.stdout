test/tokenize/keywords.as:1.1-4: `and`
test/tokenize/keywords.as:2.1-3: `or`
test/tokenize/keywords.as:3.1-4: `xor`
test/tokenize/keywords.as:4.1-4: `not`
test/tokenize/keywords.as:5.1-4: `let`
test/tokenize/keywords.as:6.1-4: `def`
test/tokenize/keywords.as:7.1-8: `mut`
test/tokenize/keywords.as:8.1-5: `true`
test/tokenize/keywords.as:9.1-6: `false`
test/tokenize/keywords.as:10.1-3: `if`
test/tokenize/keywords.as:11.1-6: `while`
test/tokenize/keywords.as:12.1-5: `loop`
test/tokenize/keywords.as:13.1-6: `break`
test/tokenize/keywords.as:14.1-9: `continue`
test/tokenize/keywords.as:15.1-7: `return`
test/tokenize/keywords.as:16.1-7: `global`
test/tokenize/keywords.as:17.1-2: end
